brigade/static/scss/molecules/_message.scss

```diff
@@ -5,14 +5,21 @@
 //
 
 .message {
-  background-color: $color-gray-light;
-  border: 0;
-  border-radius: 3px;
+  background-color: $color-white;
+  border: solid $color-gray-light 1px;
   display: flex;
   flex-direction: row;
-  font-size: 14px;
+  font-size: .9em;
   line-height: 1.5em;
-  padding: 1em;
+  margin: {
+    bottom: 1rem;
+    top: 1rem;
+  }
```

Comment:
Thoughts? I find these less readable than just doing `margin: 1rem auto;` or

```scss
margin-bottom: 1rem;
margin-top: 1rem;
```

The indentation makes me lose track of the flow of the properties, since indentation can also be used for nested rules.

Reply:
This was just repeating a pattern from the v4 style-guide `card` component, but I agree with you that it's best to avoid so it doesn't look like a nested rule.